File: Fines2Model3/DAO/CalendarioDAO.cs
using System.Net;

namespace SqlOrganize.Sql.Fines2Model3
{
    public static class CalendarioDAO
    {
    
        public static IEnumerable<Calendario> Calendarios()
        {
            string sql = "SELECT id FROM calendario ORDER BY anio DESC, semestre DESC";

            return Context.db.CacheSql().QueryIds<Calendario>("calendario", sql);
        }
    }

}

File: src/Fines2/DataAccess/CalendarioDAO.php
<?php

namespace Fines2\DataAccess;

use SqlOrganize\Sql\DbMy;
class CalendarioDAO
{

    public static function calendarios(): array{
        $db = \App\Context::getFinesDb();

        $dataProvider = $db->CreateDataProvider();

        $sql = "SELECT id
                FROM calendario 
                ORDER BY anio DESC, semestre DESC";
        return $dataProvider->fetchAllEntitiesBySqlId("calendario", $sql);
    }
}